src/component/menu/Header/Header.tsx

```diff
@@ -151,6 +184,20 @@ const Header: VFC = () => {
                         />
                     </div>
                     <div className={styles.userContainer}>
+                        <ConditionallyRender
```

Comment:
Would be nice to have three choices (light/dark/system). Not sure if necessary for the first version though.